products/redhat-openjdk.md

```diff
@@ -0,0 +1,126 @@
+    latest: "1.8.0.372.b07-3"
+    latestReleaseDate: 2023-04-27
+
+-   releaseCycle: "rhel6-8"
+    releaseLabel: "OpenJDK 8 (1.8) - RHEL 6"
+    releaseDate: 2014-10-14 # RHEL 6.6
+    eol: 2020-11-30
+    # The last release RHEL6 was 1.8.0_275 as per the docs
+    # filtered to find the latest in the series as per
+    # https://access.redhat.com/downloads/content/java-1.8.0-openjdk/x86_64/package-latest
+    latest: "1.8.0.275.b01-1"
+    latestReleaseDate: 2020-11-06
+
+-   releaseCycle: "7"
+    releaseLabel: "OpenJDK 7 (1.7)"
+    releaseDate: 2013-01-07 # RHEL 5.9
+    eol: 2020-06-30
+    # https://access.redhat.com/downloads/content/java-1.7.0-openjdk/x86_64/package-latest
+    latest: "1.7.0.261-2.6.22.2"
+    latestReleaseDate: 2020-04-15
+
+-   releaseCycle: "6"
+    releaseLabel: "OpenJDK 6 (1.6)"
+    releaseDate: 2009-01-20 # RHEL 5.3
+    eol: 2016-12-31
+    # https://access.redhat.com/downloads/content/java-1.6.0-openjdk/x86_64/package-latest
+    latest: "1.6.0.41-1.13.13.1"
+    latestReleaseDate: 2017-01-04
+
+---
+
+> [Red Hat OpenJDK](https://access.redhat.com/products/openjdk/) is a build of the Open Java Development Kit (OpenJDK)
+> with long-term support and patches from Red Hat. It is distributed as part of Red Hat Enterprise Linux,
+> but is also available with [OpenShift](https://www.redhat.com/en/technologies/cloud-computing/openshift)
+> and [Red Hat Middleware](https://developers.redhat.com/middleware).
+
+This page covers the Red Hat OpenJDK production packages, which require a
+[valid entitlement](https://access.redhat.com/articles/1299013#OpenJDK_Entitlements).
+[Developer Builds](https://developers.redhat.com/products/openjdk/download) are released
+for personal use under the [Red Hat Developer Subscription](https://developers.redhat.com/terms-and-conditions)
+and are unsupported.
+
+Red Hat documents [the differences between Red Hat OpenJDK and Oracle JDK](https://access.redhat.com/solutions/2489791).
+
+## Support
+
+A major version of OpenJDK is supported for at _minimum a period of six years_
+from the time it is first introduced by Red Hat. OpenJDK versions can fall
+out of support on versions of RHEL if the retirement date of the underlying
+RHEL platform precedes the retirement date of the OpenJDK version. There are
+currently three active major versions available in RHEL, but customers are
+advised to migrate to the newest version of OpenJDK as soon as practical to
+continue to receive updates and support.
+
+- OpenJDK 11 will receive a review later in its lifecycle, and
+the lifetime may be extended beyond 2024, if there is sufficient demand.
+
+- OpenJDK 8 is [not supported on RHEL6](https://access.redhat.com/articles/4997301),
+even with Extended Life Cycle Support(ELS) Add-on for RHEL 6.
+
+- Red Hat will deliver four updates per year, approximately three months apart,
+for the OpenJDK 8, 11, and 17 distributions.
+
+- Supported architectures for OpenJDK 8 and 11 and packaged in RPMs.
```

Comment:
No information on 17 ? This could be moved, in a more compact / less detailed form, to the description (like it was for Temurin, Coretto and other OpenJDK builds).